GraphBulkImporter/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿namespace GraphBulkImporter
+{ 
+    using System;
```

Comment:
Some files have usings inside namespace but this one has it inside namespace. Since we typically keep only one namespace per file anyways I would say you should pull these out and keep that consistency with the other files, but whichever is your preference it would be better if they were all the same!